sgl-kernel/csrc/cpu/topk.cpp

```diff
@@ -157,6 +168,103 @@ inline void sigmoid(float* __restrict__ out, const scalar_t* __restrict__ input)
   }
 }
 
+template <typename scalar_t, int SIZE>
+inline void _copy_and_convert_to_float(float* __restrict__ out, const scalar_t* __restrict__ input) {
```

Comment:
you can use at::vec::convert<scalar_t, float>, it is vectorized.

Reply:
refined accordingly